Maskbook/Scene/Wallet/Guide/GuideView.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  GuideView.swift
+//  Maskbook
+//
+//  Created by Hugo L on 2022/2/25.
+//  Copyright © 2022 dimension. All rights reserved.
+//
+
+import SwiftUI
+
+struct GuideView: View {
```

Comment:
By adding `Hashable` to `GuideItemView.Page`, the following code will  fit the demand.
If we need a `PageControl`, the page property which has been attributed by `@State` will help.

```
struct GuideView: View {
    @State
    private var page = GuideItemView.Page.one

    var body: some View {
        TabView(selection: $page) {
            ForEach(GuideItemView.Page.allCases, id: \.self) { page in
                GuideItemView(page: page)
                    .tag(page)
            }
        }
        .tabViewStyle(.page(indexDisplayMode: .never))
        .withBlueGradient(radius: 0, colorDecorator: .guideGradient)
        .ignoresSafeArea()
        .colorScheme(.light)
    }
}
```